src/main/scala/io/circe/yaml/Printer.scala

```diff
@@ -101,17 +69,60 @@ final case class Printer(
         new SequenceNode(
           Tag.SEQ,
           arr.map(jsonToYaml).asJava,
-          if (sequenceStyle == FlowStyle.Flow) DumperOptions.FlowStyle.FLOW else DumperOptions.FlowStyle.BLOCK
+          if (sequenceStyle == FlowStyle.Flow) common.FlowStyle.FLOW else common.FlowStyle.BLOCK
         ),
       obj => convertObject(obj)
     )
   }
 }
 
 object Printer {
+  final case class Default(
+    preserveOrder: Boolean = false,
+    dropNullKeys: Boolean = false,
+    indent: Int = 2,
+    maxScalarWidth: Int = 80,
+    splitLines: Boolean = true,
+    indicatorIndent: Int = 0,
+    tags: Map[String, String] = Map.empty,
+    sequenceStyle: FlowStyle = FlowStyle.Block,
+    mappingStyle: FlowStyle = FlowStyle.Block,
+    stringStyle: StringStyle = StringStyle.Plain,
+    lineBreak: LineBreak = LineBreak.Unix,
+    explicitStart: Boolean = false,
+    explicitEnd: Boolean = false
+  ) extends Printer(
+        stringStyle,
+        preserveOrder,
+        dropNullKeys,
+        mappingStyle,
+        sequenceStyle,
+        DumpSettings
+          .builder()
+          .setIndent(indent)
+          .setWidth(maxScalarWidth)
+          .setSplitLines(splitLines)
+          .setIndicatorIndent(indicatorIndent)
+          .setTagDirective(tags.asJava)
+          .setDefaultScalarStyle(StringStyle.toScalarStyle(stringStyle))
+          .setExplicitStart(explicitStart)
+          .setExplicitEnd(explicitEnd)
+          .setBestLineBreak {
+            lineBreak match {
+              case LineBreak.Unix    => "\n"
+              case LineBreak.Windows => "\r\n"
+              case LineBreak.Mac     => "\r"
+            }
+          }
+          .build()
+      )
+
+  class StreamToStringWriter extends StringWriter with StreamDataWriter {
+    override def flush(): Unit = super.flush()
```

Comment:
As this class does nothing but call the class it subclasses, it seems unnecessary.

Reply:
Good point, removed :+1: